src/lib/hasFieldSelection.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable require-yield */
+
+import {
+  FieldNode,
+  visit,
+  FragmentDefinitionNode,
+  BREAK,
+  GraphQLResolveInfo,
+} from "graphql"
+
+export const hasFieldSelection = (
+  resolveInfo: GraphQLResolveInfo,
+  match: (nodeName: string) => boolean
+): boolean => {
+  if (!resolveInfo.fieldNodes) return false
```

Comment:
Why would this not exist? Is that just for tests?

Reply:
no, i actually got this from one of the places where it was checking it and removing it was failing the test, I can remove and see which test breaks :)